custom_components/thermal_comfort/const.py

```diff
@@ -9,3 +9,7 @@
 
 DEFAULT_NAME = "Thermal Comfort"
 UPDATE_LISTENER = "update_listener"
+
+# Default values
+POLL_DEFAULT = False
+SCAN_INTERVAL_DEFAULT = 30
```

Comment:
They are part of the sensor domain. Core is moving constants from const.py to the corresponding integrations/domains e.g. SensorDeviceClass. Const should only contain stuff that isn't domain specific.